_maps/shuttles/emergency_cere.dmm

```diff
@@ -1,2122 +0,0 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
ill go ahead and put it back, but why... it's literally the biggest shuttle ever and probably doesnt even fit in most shuttle docks